### DISCUSSION TITLE
Welcome to CommandKit Discussions!

### DISCUSSION THREAD

**Discussion by notunderctrl** [neplextech/commandkit#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by ghost** [comment]
Hi
How do I make a warning system

**Reply by notunderctrl** [reply]
Unfortunately Discord.js bot features is not something we discuss in this section of the GitHub repository. This section is more for discussing the CommandKit library. For Discord.js help I suggest joining our [Discord community](https://ctrl.lol/discord).

**Reply by ghost** [reply]
ok

**Comment by ryan-hmd** [comment]
Hello, 
Is there any way to customize the warning message that CommandKit sends when the user does not have permission to use a SlashCommand?

**Reply by notunderctrl** [reply]
Yes. Kindly check issue [#69](https://github.com/underctrl-io/commandkit/issues/69)
